### PR TITLE
Closes #134: IOP test data update

### LINKED ISSUE #134: Feature Request:  Update `OE` IOP test data to include pre and post-dose IOP
### Feature Idea

See [here](https://github.com/pharmaverse/admiralophtha/issues/259)

### Relevant Input

_No response_

### Relevant Output

_No response_

### Reproducible Example/Pseudo Code

_No response_

### PR DESCRIPTION
**Pull request by ukey1025**
Thank you for your Pull Request! We have developed this task checklist from the [Development Process Guide](https://pharmaverse.github.io/admiraldev/devel/articles/development_process.html) to help with the final steps of the process. Completing the below tasks helps to ensure our reviewers can maximize their time on your code as well as making sure the admiral codebase remains robust and consistent.   

Please check off each taskbox as an acknowledgment that you completed the task or check off that it is not relevant to your Pull Request. This checklist is part of the Github Action workflows and the Pull Request will not be merged into the `devel` branch until you have checked off each task.

- [x] Place Closes #<insert_issue_number> into the beginning of your Pull Request Title (Use Edit button in top-right if you need to update)
- [x] Code is formatted according to the [tidyverse style guide](https://style.tidyverse.org/). Run `styler::style_file()` to style R and Rmd files
- [x] Updated relevant unit tests or have written new unit tests, which should consider realistic data scenarios and edge cases, e.g. empty datasets, errors, boundary cases etc. - See [Unit Test Guide](https://pharmaverse.github.io/admiraldev/devel/articles/unit_test_guidance.html#tests-should-be-robust-to-cover-realistic-data-scenarios)
- [x] If you removed/replaced any function and/or function parameters, did you fully follow the [deprecation guidance](https://pharmaverse.github.io/admiraldev/devel/articles/programming_strategy.html#deprecation)?
- [x] Update to all relevant roxygen headers and examples, including keywords and families. Refer to the [categorization of functions](https://pharmaverse.github.io/admiraldev/devel/articles/programming_strategy.html#categorization-of-functions) to tag appropriate keyword/family.
- [x] Run `devtools::document()` so all `.Rd` files in the `man` folder and the `NAMESPACE` file in the project root are updated appropriately
- [x] Address any updates needed for vignettes and/or templates
- [x] Update `NEWS.md` if the changes pertain to a user-facing function (i.e. it has an `@export` tag) or documentation aimed at users (rather than developers)
- [x] Build admiral site `pkgdown::build_site()` and check that all affected examples are displayed correctly and that all new functions occur on the "[Reference](https://pharmaverse.github.io/admiral/cran-release/reference/index.html)" page. 
- [x] Address or fix all lintr warnings and errors - `lintr::lint_package()`
- [x] Run `R CMD check` locally and address all errors and warnings - `devtools::check()`
- [x] Link the issue in the Development Section on the right hand side.
- [x] Address all merge conflicts and resolve appropriately 
- [x] Pat yourself on the back for a job well done! Much love to your accomplishment!

### CODE REVIEW

File: data-raw/oe_ophtha.R [hunk 1]
```diff
@@ -102,19 +102,32 @@ oe34 <- bind_rows(
...
-oe4 <- bind_rows(oe31, oe32, oe33, oe34) %>%
+oe41 <- bind_rows(oe31, oe32, oe33, oe34) %>%
   arrange(STUDYID, USUBJID, VISITNUM, OEDTC, OETESTCD, OELAT) %>%
   group_by(STUDYID, USUBJID) %>%
   mutate(
     "OESEQ" = row_number(),
     "DOMAIN" = "OE",
     "OEDY" = as.numeric(as.Date(OEDTC) - as.Date(RFSTDTC)) + (as.Date(OEDTC) >= as.Date(RFSTDTC)),
-    "OETPT" = NA_character_,
+    "OETPT" = PRE-DOSE,
```
Comment (manciniedoardo) [inline data-raw/oe_ophtha.R:112]: Missing quotation marks here

File: data-raw/oe_ophtha.R [hunk 2]
```diff
@@ -102,19 +102,32 @@ oe34 <- bind_rows(
...
   )
 
+oe42 <- bind_rows(oe31, oe32, oe33, oe34) %>%
+  arrange(STUDYID, USUBJID, VISITNUM, OEDTC, OETESTCD, OELAT) %>%
+  group_by(STUDYID, USUBJID) %>%
+  mutate(
+    "OESEQ" = row_number(),
+    "DOMAIN" = "OE",
+    "OEDY" = as.numeric(as.Date(OEDTC) - as.Date(RFSTDTC)) + (as.Date(OEDTC) >= as.Date(RFSTDTC)),
+    "OETPT" = POST-DOSE,
```
Comment (manciniedoardo) [inline data-raw/oe_ophtha.R:123]: Missing quotation marks here too

File: data-raw/oe_ophtha.R [hunk 3]
```diff
@@ -102,19 +102,32 @@ oe34 <- bind_rows(
...
+    "OETPTNUM" = NA
+  )
+
+oe43 <- bind_rows(oe41, oe42)
```
Comment (manciniedoardo) [inline data-raw/oe_ophtha.R:127]: This implementation duplicates all the pre-dose results to make post-dose results. Could you please add some code to randomise the post-dose values too? Otherwise all the post-dose values are identical to the pre-dose ones.

**Review by manciniedoardo — CHANGES_REQUESTED** [review]
Thanks @ukey1025, I left some comments. Could you please also add an entry to news.md?

File: data-raw/oe_ophtha.R (hunk 3 above)
Comment (ukey1025) [reply data-raw/oe_ophtha.R:130]: I have updated it - re-randomise the post-dose value. Thanks.

File: NEWS.md [hunk 4]
```diff
@@ -12,7 +12,8 @@
     - `qs_metabolic`
     - `vs_metabolic`
     - `lb_metabolic`
-    
+
+- OE data was Updated to include pre and post-dose IOP: (#134)
```
Comment (manciniedoardo) [inline NEWS.md:16]:
```suggestion
- Updated OE dataset to include both pre and post-dose IOP. (#134)   
```

Also would you mind doing me a favour please and updating the item on line 7 ("removing outdated references...") so that it is under a new `## Documentation` section, under the `## New Features`? Then we are good to go.

**Review by manciniedoardo — CHANGES_REQUESTED** [review]
Looks almost perfect, just one comment about the NEWS.md

**Review by manciniedoardo — APPROVED** [review]
